lib/asciidoctor/opal_ext/file.rb

```diff
@@ -135,6 +135,46 @@ def self.read(path)
         }
         return data;
       )
+    # afx means asciidocfx
+    when 'javafx'
+      %x(
+        var data = '';
+
+        if(afx) {
```

Comment:
`afx.readAsciidoctorResource` is specific implementation defined in the Asciidoctor FX editor code base ?